components/widgets/partials/JobOffersWidgetItem.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="job-offers-widget-item">
+  <div class="job-offers-widget-item widget-item">
```

Comment:
You should delete the hover property in scss down here. Line 101.

Reply:
Yes, good point. There was another unnecessary &:hover in ProjectsWidgetItem. Removed them both.

Reply:
Nice good job!